01week/resume/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html> <head>
+        <meta charset="UTF-8">
+        <meta name="description" content="Free Web tutorials">
+        <meta name="keywords" content="HTML,CSS,XML,JavaScript">
+        <meta name="author" content="John Doe">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" type="text/css" href="./css/style.css">
+    </head>
+<body>
+    
+<title>Elvira Joubert - Resume</title>
+<h1>Elvira Joubert - Resume</h1>
+
+<h4> Austin, Texas &#9830 512 698 0964 &#9830 
+    <a href="mailto:reagent01@gmail.com?Subject=Hello%20again"target="_top">Send Mail</a> 
+    &#9830 www.linkedin.com/in/ekjoubert</h4>
+    <h3><i> Project Management/Operations and Web Development</i> </h3>
+    <div> 
+        <p>Experience in all aspects of administrative operations, support, customer relations, mediation and research development. </p>
+    </div> 
+<h4>Career Highlights</h4> 
+<p>Launched a medical clinic for a local oil field workers and expatriates on a remote location. Translated documentation (English, Russian), provided adapted manuals for locals and expatriates; performed synchronized interpreter service at meetings  as well as tradeshows and negotiations.</p>
+<p>Reduced downtime on blue prints production for potential bidders due to the project being in a combat zone, assisted Engineering Team to support the mission. Reduced time consuming field trips to survey a site by working directly with USACE and obtaining latest satelite images to be incorporated in 2D drawings.</p>
+<p>Proposed/negotiated with Executive Management the strategic plan of providing outstanding customer service team in the aspects of dealing with creditors and opposing counsels as well as improving a working partnership with a customer. Counseled/guided customers through the debt settlement process and provided possible outcomes that allowed them to make a decision regarding the legal case. </p>
+<h4>Experience</h4>
+<ul>
+
+    <li>SpeakWrite 2017-present</li>
+    <p><i>Account Manager</i></p>
+    <li>World Law Processing - 2012-2015</li>
+    <p><i>Lead Legal Negotiator</i></p>
+    <li>Kellogg Brown and Root - 2004-2011</li>
+    <p><i>Sr. Operations Coordinator/CAD</i></p>
+    <li>Central Asia Services Group and Crown Energy - 2003-2004</li>
+    <p><i>Interpreter</i></p>
+    </ul>
+
+    <h4>Education</h4>
+    <ul>
+        <li>Master of Laws - Universtiy of Glasgow, Scotland UK</li>
+        <li>Bacherlor of Education - Aktobe State University, Aktobe, Kazakhstan</li>
+    </ul>
+</body>
+</html>
```

Comment:
please take a look at your code. try to find what is missing/wrong!

Reply:
Added semicolon line 15, linked the CSS.